back/app/services/anonymize_user_service.rb

```diff
@@ -1,8 +1,5 @@
 # frozen_string_literal: true
 
-# This service is invoked by cl2-admin/lib/admin_api.rb#project_template_export and it seems we need to require CSV.
-require 'csv'
```

Comment:
We could also remove the require statement from the other PR.

Reply:
Already in this PR

Reply:
Haha, yeah obvioulsy.